pywinauto/controls/uiawrapper.py

```diff
@@ -490,6 +522,13 @@ def _select(self, item = None):
         else:
             raise IndexError("item not found")
 
+    #-----------------------------------------------------------
+    def is_active(self):
+        """Whether the window is active or not"""
+        minimized = self.iface_window.CurrentWindowVisualState == \
+                       uia_defs.window_visual_state_minimized
+        return (not minimized and self.is_visible() and self.is_enabled())
```

Comment:
For native backend it's about global window state across all processes, otherwise it would be just an alias for `"exists visible enabled"` (like `"ready"`).